packages/framework/dds-interceptions/src/test/sharedDirectoryWithInterception.spec.ts

```diff
@@ -352,8 +352,13 @@ describe("Shared Directory with Interception", () => {
 			const userEmail = "test@microsoft.com";
 
 			// Interception callback for wrapping the subdirectory that adds user email to the attribution.
-			function interceptionCb(baseDirectory, subDirectory, key, value) {
-				const attributes = subDirectory.get(attributionKey(key));
+			function interceptionCb(
+				baseDirectory,
```

Comment:
Nit: can we add explicit types to the other parameters while we're updating some of them?